marketplacetests/marketplace/regions/search_results.py

```diff
@@ -25,6 +23,12 @@ def search_results(self):
         search_results = self.marionette.find_elements(*self._search_result_locator)
         return [Result(self.marionette, result) for result in search_results]
 
+    def install_searched_app(self, search_term):
```

Comment:
Also, I'm not sure we need this at all. We not use the existing `navigate_to_app` which is already used by lots of other tests to navigate to the app in question, and then install from there?